ui/lib/core/addon/components/confirm-action.hbs

```diff
@@ -40,7 +40,7 @@
           data-test-confirm-button
           disabled={{or @disabled @isRunning}}
           @icon={{if @isRunning "loading"}}
-          @color={{if (or (not @modalColor) (eq @modalColor "critical")) "critical" "primary"}}
+          @color={{this.confirmButtonColor}}
```

Comment:
Only using this getter here because modal colors don't map completely to button colors - the only state they share is `critical`. Critical modals are the only time we want to change the button color

Reply:
oh! haha - i'll do that, I think that's easier to glean than referring to the component file for the getter since it's a pretty simple one